FRONTEND/newFrontEnd/apTime/src/app/services/session.service.ts

```diff
@@ -16,6 +16,7 @@ export class SessionService {
   }
 
   getToken(): string {
+console.log(this.token);
```

Comment:
Shall we commit this change?

Reply:
forgot to remove debug statment. 
Removed.